.circleci/config.yml

```diff
@@ -254,7 +242,7 @@ jobs:
   test_build_alpine:
     executor: python38-alpine
     steps:
-      - run: apk add git gcc musl-dev libffi-dev openssl-dev
+      - run: apk add git gcc musl-dev libffi-dev openssl-dev bash
```

Comment:
?

Reply:
I pulled the test build commands into a bash script, which the Windows executor can handle.

Reply:
alpine doesn't come with `bash`, the helper script explicitly is using `/usr/bin/env bash`